src/components/Placeholder/components/PlaceholderGraphic/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement } from 'react';
+import Icon from '../../../Icon';
+import Typography from '../../../Typography';
+import { Props } from './types';
+
+
+function PlaceholderGraphic({
+    className,
+    graphic,
+}: Props): ReactElement {
+
+
+    return (
+            <div className={className}>
+                {graphic &&
+                    <Typography className="brew-Placeholder--graphic" as="p" variant="body1">
```

Comment:
The `brew-Placeholder--graphic` className should be `brew-Placeholder__graphic`.  `--` in BEM methodology denotes a modifier, while `__` denotes an element.  The graphic is an element of placeholder, therefore it should have two underscores.